### DISCUSSION TITLE
v0.3.0-beta.0

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#220] [General]
From now on any new features will get a minor version bump. If there's only bug fixes only the patch version will get a bump.

# What's new

## Chat with any number

Need to chat some stranger? Or want to quickly send something to some guy's number? We're all too tired to reach our phone, open contact, add the number, then finally send the message. You can now chat with other people without saving their number! Go to `Dashboard > Tools > Open Chat`

![image](https://user-images.githubusercontent.com/37852137/208702425-c3282603-da3e-4288-94e1-9111e3028aca.png)

## Monochrome tray icon

The default tray icon don't match your taste? Want a uniform look on your tray? Now you can change it to monochrome icon by going to `Dashboard > Settings > Tray Icon > Icon Type`

![image](https://user-images.githubusercontent.com/37852137/208716645-e7f7c208-cb32-4322-97bf-d7dadc4ab4e7.png)

PS: the tray badge would still be bright red when there's a new message

## Bug Fixes

- WhatsApp works with Chrome 60+
  Now more robust & should work with other language. Fixes #172

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.3.0-beta.0'>v0.3.0-beta.0</a>.</em>

**Comment by albakhtari** [comment]
Why is this a beta version?
Isn't it time for v1.0?

Since it's a beta version, I'll wait for the next version before updating the AUR package.
And even if I wanted to, I can't update the AUR package because the version can't contain a '-'.
It would only allow `v0.3.0.beta.0`

**Reply by daffamumtaz2361** [reply]
I updated some dependency, it could break some feature. I need help to make sure everything works correctly first.

> Isn't it time for v1.0?

I'm planning to do that after refactoring the codebase and releasing to flatpak

**Reply by albakhtari** [reply]
Okay, that makes sense

**Comment by Sadi58** [comment]
I've been using this for a couple of weeks now without any problems.
Thank you very much also for the monochrome tray icon options!
